assets/src/data/model/base.js

```diff
@@ -26,14 +25,21 @@ export const getQueryString = (
 ) => {
 	const where = whereConditions( queryData );
 	const { limit, order, orderBy } = queryData;
-	const queryArgs = {
-		limit,
-		order,
-		order_by: mapOrderBy( orderBy ),
-	};
-	let queryString = stringify(
-		pickBy( queryArgs, value => ! isUndefined( value ) ),
-	);
+	const queryParams = [];
+	if ( ! isUndefined( limit ) ) {
+		queryParams.push( `limit=${ limit }` );
+	}
+	if ( ! isUndefined( mapOrderBy( orderBy ) ) ) {
+		if ( isArray( mapOrderBy( orderBy ) ) ) {
+			for ( const field of mapOrderBy( orderBy ) ) {
+				queryParams.push( `order_by[${ field }]=${ order }` );
```

Comment:
this looks good for now, however do you think there'll ever be a case where we want the two order_by fields to be ordered differently (i.e. one ASC, and one DESC)?  If so, I wonder if we should account for that pattern.

Reply:
I thought about that as I was writing the code but couldn't think of any cases where we currently do that so was fine with letting that go as is for now.
It can always be improved upon later.